ci-operator/step-registry/ipi/conf/azuremag/ipi-conf-azuremag-commands.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+REGION="usgovvirginia"
```

Comment:
please do not make your region as hard code, read it from LEASED_RESOURCE, otherwise, the resource on the platform will be out of control, e.g. your step/chain is triggered 20 times in a short time, 20 clusters will be installed on the same region.

Reply:
I saw you'd added region slice in boskos in https://github.com/openshift/release/pull/24789, so just use `REGION="${LEASED_RESOURCE}"` here